plugins/modules/aci_system_connectivity_preference.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, David Neilan (@dneilan-intel) <david.neilan@intel.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_system_connectivity_preference
+short_description: APIC Connectivity Preferences (mgmt:ConnectivityPrefs)
+description:
+- Manages Alias and Banners on Cisco ACI fabrics.
```

Comment:
Noticed copy paste from other module, could you please change description

Reply:
Good catch, addressed now.